### DISCUSSION TITLE
Multi page

### DISCUSSION THREAD

**Discussion by iabhiyaan** [codex-team/editor.js#2076] [Ideas]
Hey @talyguryn, Is there is any to achieve multipage support in editor.js ?

**Comment by talyguryn** [comment]
> Hey @talyguryn, Is there is any to achieve multipage support in editor.js ?

hey. what is the point?

**Reply by iabhiyaan** [reply]
In our system we want to implement template having multiple pages and each page should be able to add content using editor js

**Comment by Kirynn** [comment]
I'm also interested in this feature, @iabhiyaan did you have any progress with this?

**Reply by iabhiyaan** [reply]
Yes. 
@Kirynn, it might not be optimal solution but kinda works !
https://codesandbox.io/s/editor-js-multiple-pages-sntd7y

**Reply by Kirynn** [reply]
Interesting, did you do anything with locking the size of the page?

**Reply by iabhiyaan** [reply]
editorjs has Readonly feature

**Reply by iabhiyaan** [reply]
const editor = new EditorJS({
  // ...
  
  readOnly: true,

  // ...
});
